### DISCUSSION TITLE
Pinning the volume mixer tab

### DISCUSSION THREAD

**Discussion by BiletoothMC** [File-New-Project/EarTrumpet#717] [Ideas]
Hi, is there a way to pin the location of the volume mixer tab so it doesn't move? / is it possible to have this position remembered so when i start up my computer and open said window it appears in the same spot?

**Comment by BiletoothMC** [comment]
Also when moving that window it has a huge delay.

**Comment by riverar** [comment]
@BiletoothMC Thanks. We're tracking the laggy window (a Windows bug) via #349.
